AUTHORS.rst

```diff
@@ -0,0 +1,10 @@
+Authors
+=======
+
+Pavol Juhas, Christopher L. Farrow, Christopher James Wright, 
+Timothy Liu, Simon J.L. Billinge
```

Comment:
Check how Tim wants to be represented.  It is up to him but he should be consistent.

Matt, I think you are an author too.

Reply:
Done and done :)